### ISSUE TITLE
Doesn't parse .uk.com domains

### ISSUE THREAD

**Issue by KevSex** [angel333/certbot-he-hook#1]
Hey,

Had an issue whereby it wouldn't work with my .uk.com domain. 

Looks like when its trying to get the `ZONENAME_REGEX`, it returns `uk\.com`.
This occurs when the CERBOT_DOMAIN is `*.domain.uk.com`

I also had to add in a `sleep` as it wasn't picking it up within the short space of time. After this, it worked great.

Also to note, `--manual-cleanup-hook` was required to remove the record once complete, not sure if its worth updating the examples shown.

**Comment by angel333** [comment]
Indeed, the regex seems to only support second level domains.

I'm afraid I'll not be able to find time to fix it now. In the meantime, however, I recommend that you try [acme.sh](https://github.com/Neilpang/acme.sh) for which I also wrote a HE plugin. It's much more polished and I'm pretty sure I've tested domains of deeper levels (HE supports adding zones for subdomains). I actually plan to incorporate code from the acme.sh plugin back to this certbot plugin.

I've added the forgotten `manual-cleanup-hook` into examples, thanks.
